apps/webapp/web/controllers/doc_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Webapp.DocController do
+  use Webapp.Web, :controller
+
+  def index(conn, _params) do
+    {:ok, app_home_path} = File.cwd
+    docs_path = "#{app_home_path}/docs"
+    {:ok, list_of_docs_name} = File.ls(docs_path)
+    list_of_docs =  Enum.map(list_of_docs_name, fn(filename) -> %{title: doc_title(filename), filename: "#{filename}"} end)
+    json conn, %{data: list_of_docs}
+  end
+
+  defp doc_title(doc_name) do
```

Comment:
Please move private functions `defp ...`  at the end of the module after all public functions.